CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+### Fixed
+
+- Fixed `test_src_set_invalid_format` not working with Wagtail 5.2 and above. ([#378](https://github.com/torchbox/wagtail-grapple/pull/378)) @JakubMastalerz
```

Comment:
**Nitpick:** No need to include the word "Fixed" within a section where everything "fixes" something.